### DISCUSSION TITLE
CORS error after few successful requests

### DISCUSSION THREAD

**Discussion by kartikshetty0897** [miguelgrinberg/Flask-SocketIO#1847] [Q&A] [answered]
I am trying to build a sign language detection webapp using Angular and Flask Socket IO. I have set the origin to "*" but after some successful requests the, I get the error " No 'Access-Control-Allow-Origin' header is present on the requested resource.". I have attached the screenshots of the response. The cors configuration that I am using is 

```
app = Flask(__name__)
app.config["SECRET_KEY"] = "mysecret"
socketio = SocketIO(app, cors_allowed_origins="*

```
Please help

![img1](https://user-images.githubusercontent.com/48761635/178958679-8de2c0c0-f2e7-4f3e-83eb-496f36b461f7.jpg)
![img2](https://user-images.githubusercontent.com/48761635/178958716-1e7f2ecb-c54b-4284-8a0b-24b6d6e0e704.jpg)
![img3](https://user-images.githubusercontent.com/48761635/178958747-34b3a203-7318-400e-9987-8e6b49c36446.jpg)

**Comment by miguelgrinberg** [comment, marked as answer]
The CORS issue is downstream of the 400 and 500 errors that you are seeing. Do you know what those are about?

**Reply by kartikshetty0897** [reply]
No idea about the 500 response, but the 400 response is returning "Invalid Session"

**Reply by miguelgrinberg** [reply]
Well, that is the problem, not CORS.

**Reply by kartikshetty0897** [reply]
Thanks Miguel. The cors error was automatically resolved after handling the problem causing 400 response
